source/opencascade/utilities.cc

```diff
@@ -328,9 +328,13 @@ namespace OpenCASCADE
       }
 
     StlAPI_Writer writer;
-    const auto    error = writer.Write(shape_to_be_written, filename.c_str());
+#  if (OCC_VERSION_MAJOR >= 7)
```

Comment:
Is `OCC_VERSION_MAJOR` available for opencascade and the oce community edition?

Reply:
I have the OpenCASCADE Community Edition CAE version 0.17 in my computer and in the detailed.log of deal.II I get.
```
OPENCASCADE_VERSION = 6.8.0
```
I checked and `OCC_VERSION_MAJOR` is available in the oce comunity edition library of my computer.

Do you know what is the difference between the OCE Comunity Edition and Open CASCADE Technology? Both libraries are available in Debian.